### DISCUSSION TITLE
QDate

### DISCUSSION THREAD

**Discussion by igormaraujo** [quasarframework/quasar#8887] [General - Components / Directives / etc]
Would be nice if the QDate component has a _header_buttons_ slot (the same place where the today button goes). I'm developing an app with quasar, and I'm using the QDate with range and landscape properties. The blue header at the left side has too many space that it is not able to use. With a slot, I will be able to create custom buttons with specific periods like '7 days' that fill the model start date 7 days ago and the end date today.

<img width="434" alt="Screen Shot 2021-04-13 at 18 28 55" src="https://user-images.githubusercontent.com/28158542/114623884-3d202680-9c86-11eb-8060-5d23787c8adc.png">

**Comment by ontwikkelfabriek** [comment]
@igormaraujo 
I would post this as a feature request in the issues page.
